content/admin/teamserver/config/settings/Disable-superadmin-access.md

```diff
@@ -0,0 +1,31 @@
+<!--
+title: "Disable SuperAdmin Access for EOP Nodes"
+description: "EOP guide for disabling SuperAdmin accounts"
+tags: "SuperAdmin EOP disable permissions"
+-->
+
+The following steps lead you through the process of disabling SuperAdmin access from Enterprise-on-Premises (EOP) nodes that are accessible to general users.  
+
+##Disable Access for Specific EOP Nodes
+
+To disable SuperAdmin access from generally accessible nodes, begin by running Contrast in a [distributed setup](https://docs.contrastsecurity.com/admin_tsinstall.html#dist) as a new installation. You'll only be prompted to disable SuperAdmin if you're installing as an *application-only installation*; a full installation won't have this option.
+
+If you already have a distributed Contrast setup, add `-Dsuper.admin.disabled=(false or true)` to `$CONTRAST_HOME/bin/contrast-server.vmoptions`. If false, the node allows SuperAdmin logins. If true, the node doesn't allow SuperAdmin logins. 
+
+>**Note:** Contrast supports one secret node.
+
+##Disable Access for SAML 
```

Comment:
I'm not sure how deep we'd would want to go. Maybe in the SAML docs we can say if you are trying to configure saml with a secret node instance you will be provided with two sets of metadata? One for the public nodes and one for the secret node. I can send you some screenshots @kaitlynmortimer of that page if you like.